### PR TITLE
added deprecated tag

### PR DESCRIPTION
**Pull request by sarab96**
added deprecated tag as discussed with Sonal

### CODE REVIEW

File: src/runtime_src/core/include/xrt.h [hunk 1]
```diff
@@ -634,6 +634,7 @@ XCL_DRIVER_DLLESPEC int xclGetBOProperties(xclDeviceHandle handle, xclBufferHand
  * This API is deprecated and will be removed in future release.
  * New clients should use xclGetBOProperties() instead
  */
+XCL_DRIVER_DLLESPEC size_t xclGetBOSize(xclDeviceHandle handle, xclBufferHandle boHandle) __attribute__ ((deprecated));
```
Comment (maxzhen) [inline src/runtime_src/core/include/xrt.h]: use XRT_DEPRECATED, instead?

File: src/runtime_src/core/include/xrt.h [hunk 2]
```diff
@@ -650,6 +651,7 @@ inline XCL_DRIVER_DLLESPEC size_t xclGetBOSize(xclDeviceHandle handle, xclBuffer
  * @boHandle:      BO handle
  * @return         uint64_t address of the BO on success
  */
+XCL_DRIVER_DLLESPEC uint64_t xclGetDeviceAddr(xclDeviceHandle handle, xclBufferHandle boHandle) __attribute__ ((deprecated));
```
Comment (maxzhen) [inline src/runtime_src/core/include/xrt.h]: use XRT_DEPRECATED, instead?

File: src/runtime_src/core/include/xrt.h [hunk 3]
```diff
@@ -634,7 +634,7 @@ XCL_DRIVER_DLLESPEC int xclGetBOProperties(xclDeviceHandle handle, xclBufferHand
  * This API is deprecated and will be removed in future release.
  * New clients should use xclGetBOProperties() instead
  */
-XCL_DRIVER_DLLESPEC size_t xclGetBOSize(xclDeviceHandle handle, xclBufferHandle boHandle) __attribute__ ((deprecated));
+XCL_DRIVER_DLLESPEC size_t xclGetBOSize(xclDeviceHandle handle, xclBufferHandle boHandle) XRT_DEPRECATED;
```
Comment (maxzhen) [inline src/runtime_src/core/include/xrt.h]: Can we move it to the front as other use cases do in this file?

File: src/runtime_src/core/include/xrt.h [hunk 4]
```diff
@@ -651,7 +651,7 @@ inline XCL_DRIVER_DLLESPEC size_t xclGetBOSize(xclDeviceHandle handle, xclBuffer
  * @boHandle:      BO handle
  * @return         uint64_t address of the BO on success
  */
-XCL_DRIVER_DLLESPEC uint64_t xclGetDeviceAddr(xclDeviceHandle handle, xclBufferHandle boHandle) __attribute__ ((deprecated));
+XCL_DRIVER_DLLESPEC uint64_t xclGetDeviceAddr(xclDeviceHandle handle, xclBufferHandle boHandle) XRT_DEPRECATED;
```
Comment (maxzhen) [inline src/runtime_src/core/include/xrt.h]: ditto

File: src/runtime_src/core/include/xrt.h (hunk 1 above)
Comment (sarab96) [reply src/runtime_src/core/include/xrt.h]: done

File: src/runtime_src/core/include/xrt.h (hunk 3 above)
Comment (sarab96) [reply src/runtime_src/core/include/xrt.h]: ?

File: src/runtime_src/core/include/xrt.h (hunk 4 above)
Comment (sarab96) [reply src/runtime_src/core/include/xrt.h]: done

File: src/runtime_src/core/include/xrt.h (hunk 3 above)
Comment (maxzhen) [reply src/runtime_src/core/include/xrt.h]: e.g.,:
XRT_DEPRECATED
XCL_DRIVER_DLLESPEC size_t xclWrite(xclDeviceHandle handle, enum xclAddressSpace space, uint64_t offset,
                                    const void *hostBuf, size_t size);

File: src/runtime_src/core/include/xrt.h (hunk 2 above)
Comment (sarab96) [reply src/runtime_src/core/include/xrt.h]: done

**Comment by maxzhen** [comment]
retest this please

**Comment by sarab96** [comment]
retest this please

**Pull request merged**